CommandHelper/LocalPackages/commands/chars/Kenpachi hero

```diff
@@ -0,0 +1,32 @@
+class: Kenpachi
```

Comment:
The file name should be
CommandHelper/LocalPackages/commands/chars/kenpachi.yml

but you have

CommandHelper/LocalPackages/commands/chars/Kenpachi hero